app/models/user.rb

```diff
@@ -56,11 +56,7 @@ def unread_notifications_count
   end
 
   def public_email
-    if additional_email.blank?
-      public_email = email
-    else
-      public_email = additional_email
-    end
+    public_email = (additional_email.blank?) ? email : additional_email
```

Comment:
Is the `public_email = ` really needed here?
I'd try to remove it.
Still much better, though.